app/editor/src/store/hooks/lookup/useSettings.ts

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { toast } from 'react-toastify';
+import { Settings } from 'tno-core';
+
+import { useLookup } from './useLookup';
+
+export interface ISettings {
+  isReady: boolean;
+  commentaryActionId?: number;
+  topStoryActionId?: number;
+  featuredStoryActionId?: number;
+  alertActionId?: number;
+}
+
+/**
+ * Provides a helper to centralize configuration setting values.
+ * @param validate Whether to validate the settings configurations.
+ * @returns Configuration setting values.
+ */
+export const useSettings = (validate?: boolean) => {
```

Comment:
This is the most important change.  It centralizes the logic to get specific setting values.  We need to start using this hook instead of writing the same code in multiple locations.